reconcile/test/test_openshift_namespaces.py

```diff
@@ -116,3 +121,17 @@ def testDeleteAbsentNamespace(self):
         oc = self.oc_clients[c1]
         oc.delete_project.assert_not_called()
         oc.new_project.assert_not_called()
+
+    def testErrorHandlingProjectExists(self):
+        oc = self.oc_clients.setdefault(c1, Mock(name=f'oc_{c1}'))
+        oc.project_exists.side_effect = StatusCodeError("SomeError")
+        self.oc_map.get.return_value = oc
+
+        self.test_ns = [
+            NS(c1, "project_raises_exception", delete=True, exists=False),
+        ]
+        f = io.StringIO()
+        with self.assertRaises(SystemExit), contextlib.redirect_stderr(f):
+            openshift_namespaces.run(False, thread_pool_size=1)
+
+            self.assertTrue("SomeError" in f.getvalue())
```

Comment:
Please use `self.assertIn` instead:
```python
self.assertIn("SomeError", f.getvalue())
```
so that when this test fails we get `f.getvalue()` in the error message and have an easier time fixing it.